test/controllers/api/validate_token_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './test_base'
+
+class API::PingControllerTest < API::TestBase
+  test "setup should return 401 with incorrect token" do
+    get api_validate_token_path, as: :json
+    assert_response 401
```

Comment:
I was originally thinking 403 but you're right. It needs to be a 401 because if your token is invalid we don't know who you are. Nice!